core/embed/prodtest/main.c

```diff
@@ -528,15 +541,28 @@ int main(void) {
 #ifdef USE_BUTTON
   button_init();
 #endif
-#ifdef USE_TOUCH
+#ifdef USE_I2C
   i2c_init();
+#endif
+#ifdef USE_TOUCH
   touch_init();
 #endif
 #ifdef USE_SBU
   sbu_init();
 #endif
   usb_init_all();
 
+#ifdef USE_OPTIGA
+  optiga_init();
+  optiga_open_application();
+  pair_optiga();
+#endif
+
+  display_reinit();
```

Comment:
do we need the second `display_reinit()`, is it a leftover, or a rebase glitch? @TychoVrahe

Reply:
leftover or rebase glitch, definitely not needed.

Reply:
removed & force pushed